ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ResponseOutputStream.java

```diff
@@ -41,7 +45,7 @@ public void write(final int b) {
   }
 
   @Override
-  public void write(final @NotNull byte[] bytes, final int offset, final int length) {
+  public synchronized void write(final @NotNull byte[] bytes, final int offset, final int length) {
```

Comment:
`ResponseOutputStream` is only written to by a single thread anyway, right? (Trying to check my understanding.)

Reply:
Yes, but it's a sanity check.